docs/build-intercept.rst

```diff
@@ -16,6 +16,51 @@ In this section, we describe how to build UnifyFS with I/O interception.
 
 .. _build-label:
 
+---------------------------------------
+UnifyFS Build Configuration Options
+---------------------------------------
+
+Fortran
+*******
+
+To enable UnifyFS use with Fortran applications, pass the ``--enable-fortran``
+option to configure. Note that only GCC Fortran (i.e., gfortran) is known to
+work with UnifyFS.
+
+GOTCHA
+******
+
+GOTCHA is the preferred method for I/O interception with UnifyFS, but it is not
+available on all platforms. If GOTCHA is not available on your target system,
+you can omit it during UnifyFS configuration by using the ``--without-gotcha``
+configure option. Without GOTCHA, static linker wrapping is required for I/O
+interception.
+
+PMI2/PMIx Key-Value Store
+*************************
+
+When available, UnifyFS uses the distributed key-value store capabilities
+provided by either PMI2 or PMIx. To enable this support, pass either
+the ``--enable-pmi`` or ``--enable-pmix`` option to configure. Without
```

Comment:
Clarifying question for my sake. When either of these is enabled, is the `-S`/`--shared-dir` option still required for `unifyfs start`? If so, what should the user provide for that option?

Or are these two things unrelated?

Reply:
They are related. That last sentence in the description is the key point. With PMI2/PMIx, you pass `--shared-dir`, and it will only be used as a backup if the PMI calls fail for some reason. If you don't have PMI2/PMIX, then we MUST have the shared filesystem directory. In either case, the usage of `unifycr start` requires that you provide the option.

Reply:
Got it! Thank you!